cumulus/polkadot-parachain/src/chain_spec/coretime.rs

```diff
@@ -85,6 +104,12 @@ impl CoretimeRuntimeType {
 
 	pub fn load_config(&self) -> Result<Box<dyn ChainSpec>, String> {
 		match self {
+			CoretimeRuntimeType::Kusama => Ok(Box::new(GenericChainSpec::from_json_bytes(
+				&include_bytes!("../../chain-specs/coretime-kusama.json")[..],
+			)?)),
+			CoretimeRuntimeType::Polkadot => Ok(Box::new(GenericChainSpec::from_json_bytes(
+				&include_bytes!("../../chain-specs/coretime-polkadot.json")[..],
+			)?)),
```

Comment:
Maybe just `todo!("Generate chainspec")` here instead of adding the dummy chain specs? WDYT?

Reply:
ok, fixed:
https://github.com/paritytech/polkadot-sdk/pull/3961/commits/515b31e967a9f06b2809c9523950c210069b9e67
https://github.com/paritytech/polkadot-sdk/pull/3961/commits/9aeabc9f6be1ad0773bb71894ebe72fdfc3ef7e9